### PR TITLE
fixed measurement path bug

### PR DESCRIPTION
**Pull request by Yantom1**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

File: examples/text-generation/quantization_tools/unify_measurements.py [hunk 1]
```diff
@@ -28,8 +28,9 @@ def unify_measurements(
 
     # save all the jsons paths in the given measurement group
     for measurement in measurement_group:
-        measurement_path = find_measurement_path(measurement, measurements_dir_path, scales, groups_size)
-        measurements_paths.append(measurement_path)
+        measurement_path = find_measurement_path(measurement, measurements_dir_path, groups_size)
+        if measurement_path is not None:
+            measurements_paths.append(measurement_path)
```
Comment (libinta) [inline examples/text-generation/quantization_tools/unify_measurements.py:33]: what should we do if the measurement_path is None?

Comment (Yantom1) [reply examples/text-generation/quantization_tools/unify_measurements.py:33]: It won't add the path and will not iterate over it in next steps. we shouldn't do anything

Comment (libinta) [reply examples/text-generation/quantization_tools/unify_measurements.py:33]: @Yantom1  it seems you didn't change the function find_measurement_path. can you update that on line 9 def find_measurement_path(measurement, measurements_dir_path, scales, group_size): ?

**Comment by HolyFalafel** [comment]
@libinta see here:
https://github.com/huggingface/optimum-habana/pull/1496

File: examples/text-generation/quantization_tools/unify_measurements.py (hunk 1 above)
Comment (libinta) [reply examples/text-generation/quantization_tools/unify_measurements.py:33]: seems this needs PR1496

**Pull request closed without merging**